api/all-features.txt

```diff
@@ -1,41 +1,54 @@
 impl core::clone::Clone for hex_conservative::Case
 impl core::clone::Clone for hex_conservative::HexToArrayError
 impl core::clone::Clone for hex_conservative::HexToBytesError
+impl core::clone::Clone for hex_conservative::OddLengthStringError
```

Comment:
What the hell? The previous commit didn't add `Clone`.

Reply:
Innnteresting.

So, previously `OddLengthStringError` was not actually exposed except as the inner type of a variant of `HexToBytesError`. Rust did not complain about this because it is a `pub struct`, it was just in a non-`pub` module. IMO this was a bug in our old API, and maybe one we could programmatically check for now that we have the API txt files.

This patch properly exports `OddLengthStringError`, and with it all the existing trait impls on it.

This is an old old issue with Rust's module system, but since it's abused to seal traits etc it's a WONTFIX.

Reply:
Oh, damn! I wish there was a first-class support for sealing so we wouldn't need to deal with this crap. Or at least some attributes for these tools to distinguish the cases. (It's not entirely WONTFIX because they could just add first-class sealing and turn those cases into deny-by-default lints. But it won't happen anytime soon.)

Reply:
This threw me as well, I believe all these additions are because the type is now available at the new location `hex_conservative::`.

Reply:
Might be worth calling it out in the commit/PR description.

Reply:
My bad, should have done that originally. Now included in the patch for the api changes.